music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_reader :on_spotify
+
+  def initialize(publish_date, on_spotify)
+    super(publish_date)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    super && @on_spotify = true
```

Comment:
The `=` operator associates a value with a variable. When using `@on_spotify = true` the value `true` will be set to the `@on_spotify` variable and it will also return `true` - making your right-hand side always `true`. Kindly use the `==` operator for comparison. 

[OPTIONAL] When evaluating a variable ruby will return the corresponding boolean value when inside a logical block. In your case, `on_spotify` will already evaluate whatever boolean value it has associated with it, so you can simply say `super && @on_spotify` instead of `super && on_spotify == true`.